JS/Lesson15/saffet/script.js

```diff
@@ -67,113 +67,348 @@ switch (day) {
-		case 4:
-		case 5:
-			console.log('Spring');
-			break;
-		case 6:
-		case 7:
-		case 8:
-			console.log('Summer');
-			break;
-		case 9:
-		case 10:
-		case 11:
-			console.log('Autumn');
-			break;
-		default:
-			console.log('Invalid Input');
+function checkScore(score){
+	if(score>0 && score<60){
+		console.log("You failed the test");
+	}
+	else if(score>60 && score<70){
+		console.log("You passed the test, but your grade is not great");
+	}
+	else if(score>71 && score<80){
+		console.log("You got a B");
+	}
+	else if(score>81 && score<90){
+		console.log("You got an A");
+	}
+	else if(score>90){
+		console.log("You got an A+ - Great job!");
+	}
+	else{
+		console.log("ınvalid input!");
 	}
 }
 
-whichMonthName(5);
-whichMonthName(10);
-whichMonthName(2);
+checkScore(50);
+checkScore(70);
+checkScore(75);
+checkScore(88);
+checkScore(93);
+checkScore(-1);
+
+console.log("-------------------------------------------------");
+// 13. Write a program that asks the user to enter a month number (1-12) and displays the corresponding season name. 
+// Use a switch statement to handle the different cases.
+console.log("Ex13:");
+function findSeason(M)
+    {
+        switch (M) {
+        case 12:
+        case 1:
+        case 2:
+            console.log("Winter");
+            break;
+        case 3:
+        case 4:
+        case 5:
+            console.log("SPRING");
+            break;
+        case 6:
+        case 7:
+        case 8:
+            console.log("SUMMER");
+            break;
+        case 9:
+        case 10:
+        case 11:
+            console.log("AUTUMN");
+            break;
+        default:
+            console.log("Invalid Month number");
+            break;
+        }
+    }
+findSeason(1);
+findSeason(3);
+findSeason(11);
+findSeason(8);
+
+console.log("-------------------------------------------------");
+// 14. Write a program that asks the user to enter a grade letter (A, B, C, D, or F) and displays a message indicating whether the grade is passing (A, B, C) or failing (D, F). 
+//Use a switch statement to handle the different cases.
+console.log("Ex14:");
+function findGrade(grade)
+    {
+        switch (grade) {
+        case "A":
+		case "a":
+        case "B":
+		case "b":	
+        case "C":
+		case "c":	
+            console.log("Passed");
+            break;
+        case "D":
+		case "d":
+        case "F":
+		case "f":
+            console.log("Failed");
+            break;
+        default:
+            console.log("Invalid Grade");
+            break;
+        }
+    }
+findGrade("A");
+findGrade("B");
+findGrade("F");
+findGrade("f");
+console.log("-------------------------------------------------");
+// 15. Write a program that asks the user to enter their age, and then calculates and displays their age in dog years (where one human year is equivalent to seven dog years). 
+//If the user enters a negative number or zero, display an error message.
+console.log("Ex15:");
+
+function dogYears(age2){
```

Comment:
Try to use verbs in the names of the functions as they always do some kind of action
```
function convertAgeToDogYears(age){
```